iceoryx_posh/test/moduletests/test_mepoo_chunk_settings.cpp

```diff
@@ -115,61 +115,12 @@ TEST(ChunkSettings_test, CallingRequiredChunkSizeReturnsCorrectValue)
 
 // END GETTER METHOD TESTS
 
-// BEGIN EXCEEDING CHUNK SIZE TESTS
-
-TEST(ChunkSettings_test, NoCustomUserPayloadAlignmentAndTooLargeUserPayload_Fails)
```

Comment:
These tests must be added again. Wrong input parameter can still lead to an overflow.